Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -81,6 +86,15 @@ void appendInteractionRegionLayersForLayer(NSMutableArray *sublayers, CALayer *l
     }
 }
 
+static BOOL webViewIsLinkPreview(UIView *view)
+{
+    auto webView = (WKWebView *)[view _scroller].superview;
+    if (![webView isKindOfClass:[WKWebView class]])
+        return NO;
+
+    return webView._isLinkPreview;
```

Comment:
This can be written as:

```
if (auto webView = dynamic_objc_cast<WKWebView>([view _scroller].superview))
    return webView._isLinkPreview;

return NO;
```